vtex/faststore
# Feat: Adds `SearchProductCard` component

## What's the purpose of this pull request?

- [x] Creates `SearchProductCard` components on @faststore/components
- [x] Adapts `SearchProductCard` on @faststore/ui
- [x] Adds `SearchProductCard` documentation page on Nextra
- [x] Adjusts `Price` and `ProductCard` components

## How to test it?

1. Test the component and its documentation:
run yarn start in `app/docs` folder and checks the `SearchProductCard` component or test it using the doc preview link:
- [SearchProductCard Component](https://faststore-site-git-feat-search-product-card-fs-755-faststore.vercel.app/components/molecules/search-product-card)

2. Test the component replacement in the application:
run yarn dev in `packages/core` -> search for a product using the search input:
   - check if any suggested product is listed
   - click on a search product card and check if the link is working correctly

<img width="1414" alt="image" src="https://user-images.githubusercontent.com/3356699/219075707-73a9a187-c9b7-4490-a692-edeeea690411.png">


### Starters Deploy Preview

[WIP]
